src/Files.App/Actions/Content/Tags/OpenAllTaggedActions.cs

```diff
@@ -38,8 +38,13 @@ public async Task ExecuteAsync()
 
 			await Task.WhenAll(files.Select(file 
 				=> NavigationHelpers.OpenPath(file.path, _pageContext.ShellPage!)));
-
-			folders.ForEach(async folder => await NavigationHelpers.OpenPathInNewTab(folder.path, false));
+			
+			var openTasks = folders.Select(async folder =>
+			{
+				await NavigationHelpers.OpenPathInNewTab(folder.path, false);
```

Comment:
I think this messes with the order, the ideal behavior is for them to open in the same order they are displayed in the list.